docs/RefMan/BasicSyntax.rst

```diff
@@ -107,9 +107,51 @@ end of the line.  Block comments may be nested arbitrarily.
   // This is a line comment
   /* This is a /* Nested */ block comment */
 
-.. todo::
-  Document ``/** */``
+Documentation Comments
+----------------------
 
+Declarations in Cryptol can have documentation attached to them using special
+*docstring* comments. These attached comments are accessible in the REPL
+using the `:help` command.
```

Comment:
Given that this is ReStructured Text (rather than markdown), I believe inline text needs to be surrounded by double backticks:

```suggestion
using the ``:help`` command.
```

And similarly for other snippets of inline code below.